src/main/java/pl/touk/sputnik/review/Review.java

```diff
@@ -48,11 +52,6 @@ public Review(@NotNull List<ReviewFile> files) {
     }
 
     @NotNull
-    public List<String> getBuildDirs() {
-        return Lists.transform(files, new ReviewFileBuildDirFunction());
-    }
-
-    @NotNull
     public List<String> getSourceDirs() {
```

Comment:
Gradle reuses Maven convention so it's not needed now. Maybe when we introduce support for other tools.